tools/BcosAirBuilder/build_chain.sh

```diff
@@ -348,6 +356,33 @@ download_bin()
     chmod a+x ${binary_path}
 }
 
+download_monitor_bin()
+{
+    if [ ! -z "${mtail_binary_path}" ];then
+        LOG_INFO "Use binary ${mtail_binary_path}"
+        return
+    fi
+    if [ "${x86_64_arch}" != "true" ];then exit_with_clean "We only offer x86_64 precompiled fisco-bcos binary, your OS architecture is not x86_64. Please compile from source."; fi
+    mtail_binary_path="bin/${mtail_binary_name}"
```

Comment:
Recommend to download mtail to `~/.fisco/mtail`, refer to the tassl download path.